packages/eslint-plugin-next/src/index.ts

```diff
@@ -22,12 +24,12 @@ const recommendedRules = {
   '@next/next/no-duplicate-head': 'error',
   '@next/next/no-head-import-in-document': 'error',
   '@next/next/no-script-component-in-head': 'error',
-}
+} satisfies Record<string, SeverityString>
 
 const coreWebVitalsRules = {
   '@next/next/no-html-link-for-pages': 'error',
   '@next/next/no-sync-scripts': 'error',
-}
+} satisfies Record<string, SeverityString>
```

Comment:
Can we use `const coreWebVitalsRules: Record<string, SeverityString>` instead? We do want to support older TS versions without `satisfies`. Explicit annotation would also give auto-complete when typing new rules I believe.

Reply:
Right. And with a type annotation we loose enumeration of the rules in types.